components/public-api/gitpod/v1/workspace.proto

```diff
@@ -65,6 +72,51 @@ message ListWorkspacesResponse {
   PaginationResponse pagination = 2;
 }
 
+message CreateAndStartWorkspaceRequest {
+  message Source {
+
+    message Git {
+      string clone_url = 1;
+
+      // ref is an alternatively symbolic. e.g. refs/tags/v1.0,
+      // 9585191f37f7b0fb9444f35a9bf50de191beadc2
+      string ref = 2;
```

Comment:
add to docs: "" means checkout the default branch, which is also what happens when the ref doesn't exist